include/variables.h

```diff
@@ -15,7 +15,7 @@ extern s32   gLapCountByPlayerId[8];             // D_80164390
 extern s32   gPlayerIsThrottleActive[];           // D_801653E0
 extern s32   gGPCurrentRaceRankByPlayerId[];      // D_801643B8
 
-extern struct_8018CA70_entry D_8018CA70[];
+extern hud_player hudPlayers[];
```

Comment:
player1HUD ?

I'm not opposed to `hud_` being a prefix so long as its used consistently and the rest of the name follows camelCase:
ex: `hud_myVariable`
`hud_player1`